src/scripts/underground/Underground.ts

```diff
@@ -1,50 +1,81 @@
-/**
- * Created by dennis on 03-07-17.
- */
 class Underground {
+    public static saveKey: string = "underground";
+
     public static itemSelected;
     public static energyTick: KnockoutObservable<number> = ko.observable(60);
     public static counter: number = 0;
 
+    public static energy = ko.observable(50);
```

Comment:
As much as I love refreshing the page to refill energy, this should probably be saved and loaded :)

Reply:
😅